### DISCUSSION TITLE
Need fluent-bit spec file for release 2.0.6

### DISCUSSION THREAD

**Discussion by ashutosh-maheshwari** [fluent/fluent-bit#7006] [General]
Hello Team,
Can I please get the spec file for fluent-bit release 2.0.6?
@nokute78 @edsiper @leonardo-albertovich

**Comment by patrick-stephens** [comment]
What spec file do you mean? Everything used to make a release is in the Git repo.

**Reply by ashutosh-maheshwari** [reply]
I want to create my own rpm. So, I wanted to get the spec file that you guys use for fluent-bit. I was not able to find it in the repo.

**Reply by patrick-stephens** [reply]
We use cmake, or more specifically cpack, to produce the various packages for all targets. I suspect you could get it to spit out a spec file but you could also just use it directly.

https://github.com/fluent/fluent-bit/blob/7fd995072a0e0591888629b39619c5783a5ea047/packaging/distros/centos/Dockerfile#L166

There is a `packaging/build.sh -d <distro>` script to do it for you.
